pkg/configmap/configmap_writer.go

```diff
@@ -63,6 +64,32 @@ func TranslateInvalidChars(input string) string {
 	return validConfigMapKey
 }
 
+func validateAnnotationKeys(file AnnotationsFile) error {
+	var errors []string
+	if file.Annotations.Resources == "" {
+		errors = append(errors, fmt.Errorf("No value set for operators.operatorframework.io.bundle.manifests.v1").Error())
```

Comment:
all of these can use the constant value in pkg/lib/bundle

Reply:
@jpeeler thank you for pointing the constants out. I've updated the PR to use the constants.